### PR TITLE
Responsive first try HW lesson 9

### PR DESCRIPTION
**Pull request by sdmnc**
image

### CODE REVIEW

**Review by emma13z — COMMENTED** [review]
I don't see any media query. How did you manage to adjust responsiveness? did you make changes in the code itself?

**Comment by sdmnc** [comment]
> I don't see any media query. How did you manage to adjust responsiveness? did you make changes in the code itself?

Well, I didn't manage to adjust responsiveness fully :-) Just played with width: auto and 100%

**Comment by emma13z** [comment]
I don't think you had to change the code itself. You have to add media query in your css and there to make changes for responsiveness :)

File: index.html [hunk 1]
```diff
@@ -2,14 +2,68 @@
...
-    <link rel="stylesheet" href="style.css" <title>Kmarketer</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    <title>Kmarketer</title>
+</head>
 
 <body>
+    <section class="top-banner">
```
Comment (IvanCeban) [inline index.html:13]: Good for using BEM!

**Review by IvanCeban — APPROVED** [review]
You did the responsiveness trick with flex wrap. Good job!
But I still want you to use media queries ;)
at least in future

**Pull request merged**